mkFit/MkFinder.cc

```diff
@@ -179,18 +179,42 @@ void MkFinder::getHitSelDynamicWindows(const float invpt, const float theta, flo
   float this_dphi = (ILC.c_dp_0)*invpt+(ILC.c_dp_1)*theta+(ILC.c_dp_2);  
   // In case layer is missing (e.g., seeding layers, or too low stats for training), leave original limits
   if(this_dphi>0.f){
-    min_dphi = (ILC.c_dp_sf)*this_dphi;
+    if ((theta>0.8f && theta<1.1f) || (ILC.c_dp_sf)*this_dphi>min_dphi){
```

Comment:
here are some hardcoded numbers about a specific theta region... not ideal, but can be improved in a later PR

Reply:
As you noted, these will go in a next PR (by the end the weekend), which will include also a 'fix' of the geometry-driven dphi windows.